.github/workflows/tests-typecheck.yml

```diff
@@ -1,16 +1,14 @@
 name: Tests, Linter & Typecheck
 
-on:  
+on:
```

Comment:
The formatter in my VS Code ran on this file, which accounts for the additional whitespace changes.